Cargo.toml

```diff
@@ -123,8 +123,7 @@ tracing = "0.1.41"
 tracing-opentelemetry = { version = "=0.28.0", default-features = false }
 tracing-subscriber = { version = "0.3.19", default-features = false, features = ["fmt"] }
 twox-hash = "2.1.0"
-# TODO (WAL-790): update typed-store to >= 1.47 or remove it.
-typed-store = { git = "https://github.com/MystenLabs/sui", tag = "testnet-v1.46.0" }
+typed-store = { git = "https://github.com/MystenLabs/sui", tag = "testnet-v1.48.0" }
```

Comment:
I believe this is necessary in this release. I did not see an obvious way to avoid bumping `typed-store` this time.